Distiller-server/src/main/java/com/mapr/distiller/server/recordtypes/ThreadResourceRecord.java

```diff
@@ -84,78 +141,43 @@ public static boolean produceRecord(RecordQueue thread_resources, String produce
 	public String toString(){
```

Comment:
I didn't realize @Override was a compiler instruction.  I thought it was just a comment... I've probably been omitting that from multiple places that it should be in.